react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -18,31 +19,32 @@ import { AVATAR_SIZE } from '../styles';
 import styles from './styles';
 import VideoMutedIndicator from './VideoMutedIndicator';
 
+/**
+ * Thumbnail component's property types.
+ */
+type Props = {
```

Comment:
There were a lot of already existing props here, but can you please add docs to those that you added?